src/main/java/com/example/accountz/webController/UserController.java

```diff
@@ -0,0 +1,70 @@
+package com.example.accountz.webController;
+
+import com.example.accountz.model.UserDto;
+import com.example.accountz.security.TokenProvider;
+import com.example.accountz.service.UserService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Set;
+
+@Slf4j
+@RestController
+@RequestMapping("/auth")
+@RequiredArgsConstructor
+public class UserController {
+    private final UserService userService;
+    private final TokenProvider tokenProvider;
+
+    @PostMapping("/sign-up")
+    public ResponseEntity<?> signUp(
+            @RequestBody @Valid UserDto.SignUp request) {
+
+        this.userService.registerUser(request);
+
+        return ResponseEntity.ok().body(
+                new ApiResponse("회원가입","Success"));
+
+    }
+
+    @PostMapping("/log-in")
+    public ResponseEntity<?> signIn(
+            @RequestBody UserDto.SignIn request) {
+
+        var member =
+                this.userService.authenticateUser(request);
```

Comment:
불필요한 new line 은 최대한 없애주세요 ^^; 
```suggestion
        var member = this.userService.authenticateUser(request);
```